options.go

```diff
@@ -15,87 +15,69 @@
 package greptime
 
 import (
+	"crypto/tls"
 	"time"
 
 	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials"
 	"google.golang.org/grpc/credentials/insecure"
 	"google.golang.org/grpc/keepalive"
 )
 
 var (
-	uaOpt = grpc.WithUserAgent("greptimedb-ingester-go/" + Version)
-
-	// TODO(yuanbohan): SecurityOptions
+	uaOpt       = grpc.WithUserAgent("greptimedb-ingester-go/" + version)
+	secureOpt   = grpc.WithTransportCredentials(credentials.NewTLS(&tls.Config{InsecureSkipVerify: false}))
 	insecureOpt = grpc.WithTransportCredentials(insecure.NewCredentials())
-
-	defaultKeepaliveInterval = 30 * time.Second
-	defaultKeepaliveTimeout  = 5 * time.Second
 )
 
-type Options struct {
-	keepalive *KeepaliveOption
+type options struct {
```

Comment:
suggestion: Can we add more fields to configure client tls? For example(can be changed by a better naming):

```
type Options struct {
   ...
   InsecureSkipVerify bool
   ServerCertPath string
   ClientKeyPath string
   ClientCertPath string
   ...
}
```

- `InsecureSkipVerify`: The user can control whether to check the server cert; default can be false for security reasons. 
- `ServerCertPath`: It's needed when `InsecureSkipVerify` is `false`. If we use the trusted certificate like let's certificate, maybe it can be host root CA;
- `ClientKeyPath` and `ClientCertPath`: Use for mTLS;

There's a more complicated and detailed reference about etcd client v3: https://pkg.go.dev/go.etcd.io/etcd/client/v3#Config.

Reply:
good point. 

- `mTLS` option will not be included in this PR, and I will open an issue for it.
- `ServerCertPath` is a need field. Is it ok to support it in next PR which will make this PR simpler.

By the way, `ServerCertPath` is supported in GreptimeCloud?

Reply:
etcd client is a good project to refer Go practices!

Reply:
#13 will cover the cert and key config, and it will be resolved in the following PR